### DISCUSSION TITLE
How to disable email confirmation in local development

### DISCUSSION THREAD

**Discussion by piggy0713** [supabase/supabase#7890] [Questions] [answered]
I tried to set up a local supabase env (docker) with react. I keep getting "Error sending confirmation mail" with supabase.auth.signUp.  In the local supabase studio, there is no setting in auth to disable email confirmation. Is there any way to bypass the email confirmation in the local environment?

**Comment by GaryAustin1** [comment, marked as answer]
I don't use the local studio, but just want to verify you are looking in right place as it moved a few weeks back with a dashboard update (at least on the hosted instances...  You now have to toggle the email provider to see the options.

![image](https://user-images.githubusercontent.com/54564956/180898919-0ac4f712-a290-43f5-83ea-46401eeaa8ca.png)

**Reply by piggy0713** [reply]
thank you for reply, in local studio , there is no setting under auth
<img width="260" alt="Screen Shot 2022-07-25 at 22 05 59" src="https://user-images.githubusercontent.com/28695185/180907014-22b220b1-8cc6-4a83-ab18-16ee0e4cf4eb.png">
<img width="250" alt="Screen Shot 2022-07-25 at 22 06 23" src="https://user-images.githubusercontent.com/28695185/180907016-297ac6ad-6e19-4e53-b882-e134ee5ce003.png">

**Reply by GaryAustin1** [reply]
Sorry, I should have remembered that.  Below are a few links that might help.

Here is the gotrue config info... https://supabase.com/docs/gotrue/server/about
In that is a setting for auto confirm: "MAILER_AUTOCONFIRM - bool"
This talks about setting environment for docker:  https://supabase.com/docs/guides/hosting/overview#configuration

I did not search here.  There already may be an more direct answer to how to set the configs.

**Reply by J0** [reply]
Hey, 

Thanks for the query! @GaryAustin1 is right -- you can disable  email confirmations by setting `MAILER_AUTOCONFIRM` to false.

Let us know if there are further issues.
Joel
